ppr-ui/src/components/search/SearchBar.vue

```diff
@@ -40,7 +40,7 @@
           </v-tooltip>
         </div>
       </v-col>
-      <v-col v-if="!isStaffBcolReg && !isStaffSbc" align-self="end" cols="3">
+      <v-col align-self="end" cols="3">
```

Comment:
Was there further commits here to render for specific types of Staff or is the requirement to display for all, client and staff?

Reply:
I checked with Melissa and the field is shown in all cases.